django/db/backends/postgresql/operations.py

```diff
@@ -271,5 +272,18 @@ def explain_query_prefix(self, format=None, **options):
             prefix += ' (%s)' % ', '.join('%s %s' % i for i in extra.items())
         return prefix
 
-    def ignore_conflicts_suffix_sql(self, ignore_conflicts=None):
-        return 'ON CONFLICT DO NOTHING' if ignore_conflicts else super().ignore_conflicts_suffix_sql(ignore_conflicts)
+    def on_conflict_suffix_sql(
+        self, opts, fields, on_conflict=None, update_fields=None, unique_fields=None
+    ):
+        if on_conflict == OnConflict.Ignore:
+            return 'ON CONFLICT DO NOTHING'
+
+        elif on_conflict == OnConflict.Update:
```

Comment:
This can simply be `if` when the prior `if` block always returns or raises. There are a few more instances of this below.